docs/aws-professional/services.md

```diff
@@ -22,168 +22,168 @@ For an overview of Azure for AWS users, see [Introduction to Azure for AWS exper
-| Dedicated network             | Direct Connect         | [ExpressRoute](https://azure.microsoft.com/services/expressroute/)                                                                                          | Establishes a dedicated, private network connection from a location to the cloud provider (not over the Internet).                                                                                                                                    |
-| Load balancing                | Elastic Load Balancing | [Load Balancer](https://azure.microsoft.com/services/load-balancer/) <br/><br/>[Application Gateway](https://azure.microsoft.com/services/application-gateway/) | Automatically distributes incoming application traffic to add scale, handle failover, and route to a collection of resources.                                                                                                                         |
+| Cloud virtual networking      | Virtual Private Cloud  | [Virtual Network](https://azure.microsoft.com/services/virtual-network/)                                                                                    | Provides an isolated, private environment in the cloud. |
+| Cross-premises connectivity   | AWS VPC Gateway        | [VPN Gateway](/azure/vpn-gateway/vpn-gateway-about-vpngateways)                                                             | Connects Azure virtual networks to other Azure virtual networks or customer on-premises networks. It also supports VPN tunneling.                    |
+| Domain name system management | Route 53               | [DNS](https://azure.microsoft.com/services/dns/)                                                                                                      | Manage DNS records using the same credentials, billing, and support contract as other Azure services.                                                                                |
+| **&nbsp;**                    | Route 53               | [Traffic Manager](https://azure.microsoft.com/services/traffic-manager/)                                                                                    | Service that hosts domain names, routes users to Internet applications, manages traffic to apps, and improves app availability with automatic failover.                                                |
+| Content delivery network      | CloudFront             | [Content Delivery Network](https://azure.microsoft.com/services/cdn/)                                                                                 | Global content delivery network that transfers audio, video, applications, images, and other files.                           |
```

Comment:
"Transfer" doesn't seem to be the right term here. Serves or delivers. 

(I guess "delivers" was slightly redundant but here, but I think it's OK.)

Reply:
delivers if fine